### PR TITLE
Don't install Cloud SDK when importing Debian 8.

### PR DESCRIPTION
**Pull request by EricEdens**
The Cloud SDK is currently broken for Debian 8 imports; the SDK dropped support for Python 3.4, which is Debian 8's version.

This change skips installing the Cloud SDK for Debian 8, and skips testing for it.

## Testing

Ran the following:
1. daisy -project edens-test -zone=us-central1-b daisy_integration_tests/debian_8_translate.wf.json
2. daisy -project edens-test -zone=us-central1-b daisy_integration_tests/debian_9_translate.wf.json

### CODE REVIEW

File: daisy_integration_tests/scripts/post_translate_test.sh [hunk 1]
```diff
@@ -135,6 +135,14 @@ function check_google_cloud_sdk {
     fi
   fi
 
+    # Skip for Debian 8: b/161567418
```
Comment (zoran15) [inline daisy_integration_tests/scripts/post_translate_test.sh:138]: minor: change the bug link (which is not accessible outside Google) with a description (or have both)

File: daisy_workflows/image_import/debian/translate.py [hunk 2]
```diff
@@ -67,13 +67,15 @@ def DistroSpecific(g):
       logging.warn('Could not uninstall Azure agent. Continuing anyway.')
 
     utils.update_apt(g)
-    pkgs = ['google-cloud-packages-archive-keyring', 'google-cloud-sdk',
-            'google-compute-engine']
-    # Debian 8 doesn't support the new guest agent, so we need to install
-    # the legacy Python version.
```
Comment (zoran15) [inline daisy_workflows/image_import/debian/translate.py:73]: Any consequences for the old Guest agent and running on python 2.7/3.4?

File: daisy_integration_tests/scripts/post_translate_test.sh (hunk 1 above)
Comment (EricEdens) [reply daisy_integration_tests/scripts/post_translate_test.sh]: done!

File: daisy_workflows/image_import/debian/translate.py (hunk 2 above)
Comment (EricEdens) [reply daisy_workflows/image_import/debian/translate.py:73]: AFAIK there aren't big changes planned for the old guest agent.

**Pull request merged**